11-multi-track/bb_models_collections/readme.md

```diff
@@ -36,58 +36,68 @@ How we got here:
 4. Frameworks
   - All the structure of OOJS with the ease of writing like jQuery.
 
-What's the purpose of a front end framework? JS and all of it's many libraries are great, but you can start building and building your application and all of a sudden there's no structure and everything's soup.
+What's the purpose of a front end framework?
+> JS and all of it's many libraries are great, but you can start building your application and all of a sudden there's no structure and everything's soup.
 
-### libraries (2.5/7.5)
+### Libraries (2.5/7.5)
 
 - libraries gives us tools to utilize.
 - abstracts code and allows us to write our code more succinctly
 - allows us to write applications faster and easier
 - lots of options, very few rules (jQuery)
 
-### frameworks (2.5/10)
+### Frameworks (2.5/10)
 
 - like libraries in that it gives us tools to utilize
-- additionally they provide structure and conventions users have to follow in order for them to work.
+- additionally, they provide structure and conventions users have to follow in order for them to work.
 - Lots of rules (convention), few options (Ruby on Rails)
 
-## What is a front end framework? (5/15)
-- a library that attempts to move some or all application logic to the browser, while providing a simple interface for keeping the front-end in sync with the back-end
+## What is a Front End Framework? (5/15)
+- a library that attempts to move some or all of a application's logic to the browser, while providing a simple interface for keeping the front-end in sync with the back-end
 - applications can run completely in the browser, minimizing server load since the server is only accessed when the front end needs to synchronize data with the backend
 - server sends over the app in the initial request (HTML/CSS/JS) then JS makes all subsequent requests with AJAX
 - provides more fluid user experience
-- loads everything from the database on page load (data and templates) then renders/updates the page content based on user interaction.
+- loads everything from the database on page load (data and templates) then **renders/updates** the page content based on user interaction.
 
 
-### What is backbone? (25/40)
-Individual reading(10/25)
-- Go to [backbonejs documentation](http://backbonejs.org/)
+### What is [Backbone](http://backbonejs.org/#)? (25/40)
+**Doc Dive (10/25)**
+- Go to [backbonejs documentation](http://backbonejs.org/#Getting-started)
   - Read from Getting Started to Backbone.Model
+  - Check out some [examples](http://backbonejs.org/#examples) of Backbone in the wild
 
 ### T&T(5/30)
-- With what you know about OOJS, jot down some ideas about what you think Backbone is and why you would use it.
+With what you know about OOJS, jot down some ideas about:
+- What problem does Backbone aim to solve?
+- When or why would you use Backbone?
+- How might you use Backbone components in conjunction with other technologies
 
```

Comment:
Since they've had experience with Angular, can ask them to note comparisons / differences.